src/redirects.ts

```diff
@@ -52,6 +52,30 @@ export async function redirects(request: Request) {
         return Response.redirect('https://meet.google.com/kon-wdmt-yhb')
       case '/labschool':
         return Response.redirect('https://meet.google.com/cvd-pame-zod')
+      case '/1':
+        return Response.redirect(
+          'https://docs.google.com/document/d/1FhDqkRBZZEkxN-j9edfP-k0kdbnjurKZ3nwoSMxlilI/edit#'
```

Comment:
```suggestion
          'https://meet.google.com/fxn-iprp-ezx'
```

You shall use the meet.google.com links in the document https://docs.google.com/document/d/1FhDqkRBZZEkxN-j9edfP-k0kdbnjurKZ3nwoSMxlilI/edit#

For example /1 goes to https://meet.google.com/fxn-iprp-ezx, /2 goes to https://meet.google.com/yku-aksd-fkk etc